### PR TITLE
Update SPMT to Python 3.10 - PEP498 Literal String Interpolation

### PR DESCRIPTION
**Pull request by Liquid369**
This pull request is updating the entire codebase to PEP 498 standards using fstrings. 

This PEP proposed a new syntax for formatting strings that is more concise, readable, and less error-prone compared to other string formatting methods like str.format() or % formatting. F-strings allow you to embed expressions inside string literals by prefixing the string with f or F and enclosing the expressions within curly braces {}.

I have updated every file to this standard and tested that our application is still functioning.

We have in this the added PyQt version bumped to `5.15.10`
Then we have also set wheel and setup tools to specific versions to avoid any of our dependancies with setup.py warnings on depreciation coming.

There is a small update in the spmt.py that is not the fstrings, but for testing purposes I needed to convert the splash sizing into an integer to load. This can be removed for a different fix, or whichever.

### CODE REVIEW

File: src/mainWindow.py [hunk 1]
```diff
@@ -396,14 +395,14 @@ def saveMNListOrder(self):
...
-        myPopUp_sb(self, "info", 'SPMT - hw check', "%s" % self.hwStatusMess)
+        myPopUp_sb(self, "info", 'SPMT - hw check', f"{self.hwStatusMess}")
 
     def showRPCstatus(self, server_index, fDebug):
         # Update displayed status only if selected server is not changed
         if server_index == self.header.rpcClientsBox.currentIndex():
             self.updateRPCled(fDebug)
             if fDebug:
-                myPopUp_sb(self, "info", 'SPMT - rpc check', "%s" % self.rpcStatusMess)
+                myPopUp_sb(self, "info", 'SPMT - rpc check', f"{self.rpcStausMess}")
```
Comment (Fuzzbawls) [inline src/mainWindow.py]: typo: `rpcStausMess` -> `rpcStatusMess`

File: src/tabGovernance.py [hunk 2]
```diff
@@ -116,7 +116,7 @@ def itemButton(value, icon_num):
 
         # update MN count
         mnCount = self.caller.parent.cache['MN_count']
-        self.ui.mnCountLabel.setText("Total MN Count: <em>%d</em>" % mnCount)
+        self.ui.mnCountLabel.setText("Total MN Count: <em>{mnCount}</em>")
```
Comment (Fuzzbawls) [inline src/tabGovernance.py]: missing `f` before quoted string

**Review by Fuzzbawls — CHANGES_REQUESTED** [review]
some changes needed, see inline comments.

Also, the `timeThis` function in `src/misc.py` needs updating as newer versions of python no longer have the `time.clock()` function:
```diff
Index: src/misc.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/misc.py b/src/misc.py
--- a/src/misc.py	
+++ b/src/misc.py	
@@ -491,9 +491,9 @@
 
 def timeThis(function, *args):
     try:
-        start = time.clock()
+        start = time.time()
         val = function(*args)
-        end = time.clock()
+        end = time.time()
         return val, (end - start)
     except Exception:
         return None, None

```

File: src/mainWindow.py (hunk 1 above)
Comment (Liquid369) [reply src/mainWindow.py]: Fixed and pushed

File: src/tabGovernance.py (hunk 2 above)
Comment (Liquid369) [reply src/tabGovernance.py]: fixed and pushed

**Comment by Liquid369** [comment]
Fixed all review suggestions. 

Thanks for those catches!

**Review by Fuzzbawls — APPROVED** [review]
ACK 7d6a69dbc15a88b6b75a83e55b820ea1f2d2b474

i've tested as much as I can using this PR with the changes suggested. without a dedicated test framework this is the best we can do.

**Pull request merged**